fighthealthinsurance/settings.py

```
@@ -692,7 +694,38 @@ def EXTERNAL_STORAGE_B(self):
+                        access_key=self.BB_STORAGE_ACCESS_KEY,
+                        secret_key=self.BB_STORAGE_SECRET_KEY,
+                        region=self.BB_REGION_NAME,
+                    )
+                    minio = MinioStorage(
+                        minio_client,
+                        bucket_name=self.BB_STORAGE_MEDIA_BUCKET_NAME,
+                        auto_create_bucket=True,
+                    )
+                    return minio
+                else:
+                    raise Exception("Back blaze storage endpoint configured")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)